source/content/terminus/install.md

```diff
@@ -37,6 +37,21 @@ curl -O https://raw.githubusercontent.com/pantheon-systems/terminus-installer/ma
 
 See [Troubleshooting](#troubleshooting) if your installation fails, or the [Installation](https://github.com/pantheon-systems/terminus#installation) section of the Terminus README file on GitHub for advanced installation methods.
 
+### Windows: Enable the terminus command
+
+If you encounter the following error after installation completes:
+
+```bash
+Terminus was installed, but the installer was not able to write to your bin dir. To enable the
+`terminus` command, add this alias to your .bash_profile (Mac) or .bashrc (Linux) file:
+```
+
+Use `alias` to have the `terminus` command call the application:
+
+```bash{promptUser: user}
+alias terminus=~/vendor/bin/terminus
+```
+
```

Comment:
This change loses the reference to the `.bashrc` file, and link for source reference. Porque?

Reply:
newest commit uses `echo` to add it to the end of `.bashrc`